src/Confluent.Kafka/Consumer.cs

```diff
@@ -194,18 +194,37 @@ public void Unsubscribe()
             => consumer.Unsubscribe();
 
         /// <summary>
-        ///     Update the assignment set to \p partitions.
+        ///     Update the assignment set to <param name="partitions" />.
         ///
-        ///     The assignment set is the set of partitions actually being consumed
-        ///     by the KafkaConsumer.
-        /// </summary>
-        /// <remarks>
-        ///     blocks
-        /// </remarks>
+        ///     The assignment set is the complete set of partitions to consume
+        ///     from.
+        /// </summary>
+        /// <param name="partitions">
+        ///     The set of partitions to consume from. If an offset value of
+        ///     -1001 (Offset.Invalid) is specified for a partition, consumption
```

Comment:
My current thoughts are to leave this as is because people are not likely to want to specify this directly here now there is the overload of Assign that takes `TopicPartition`s. I may well flip-flop on that opinion, but I think this PR is a step forward so we should merge it.